frontend/src/pages/Common/Budget.js

```diff
@@ -320,10 +320,16 @@ const Budget = (props) => {
             variant="contained"
             color="secondary"
             data-test="add-projected-budget"
-            disabled={!budgetAmountAdd || !currency || !organization || !isSaveable || !isValidBudgetAmountAdd}
+            disabled={
+              !isSaveable ||
+              !isValidBudgetAmountAdd ||
+              !organization ||
+              (budgetAmountAdd && !currency) ||
+              (!budgetAmountAdd && currency)
```

Comment:
Pls change currency to !!currency, otherwise the expression evaluates to a string and there is an error in console.
```
Warning: Failed prop type: Invalid prop `disabled` of type `string` supplied to `ForwardRef(ButtonBase)`, expected `boolean`.
    at ButtonBase (http://localhost:3000/static/js/bundle.js:81820:82)
    at http://localhost:3000/static/js/bundle.js:51758:66
    at Button (http://localhost:3000/static/js/bundle.js:81521:59)
    at div
    at div
    at div
    at Budget (http://localhost:3000/static/js/bundle.js:7858:86)
    at div
    at div
    at ProjectDialogContent
    ...
   ```

Reply:
Implemented